test/system/950-auth-scripts.bats

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bats
+#
+# Tests for podman auth scripts
+#
+
+load helpers
+load helpers.network
+
+function setup() {
+    if is_remote; then
+       skip "only applicable on podman"
+    fi
```

Comment:
If the customer wants to check podman-remote the test should be done for it as well.